lib/report-portal-client.js

```diff
@@ -145,9 +152,15 @@ class RPClient {
             this.map[tempId] = this.getNewItemObj(resolve => resolve(launchDataRQ));
             this.map[tempId].realId = launchDataRQ.id;
         } else {
+            const systemAttr = helpers.getSystemAttribute();
+            const attributes = launchDataRQ.attributes != null
+                && typeof launchDataRQ.attributes[Symbol.iterator] === 'function'
```

Comment:
Can `attributes` be another iterable object type than array?
May it be just like the next line?
```
const attributes = Array.isArray(launchDataRQ.attributes) ? launchDataRQ.attributes.concat(systemAttr) : systemAttr;
```

Reply:
It makes sense but I added this check for additional security.